src/main/java/com/beanbeanjuice/command/moderation/SetDailyChannelCommand.java

```diff
@@ -0,0 +1,97 @@
+            return;
+        }
+
+        if (args.size() == 1) {
+            String commandTerm = args.get(0);
+            if (commandTerm.equalsIgnoreCase("remove") || commandTerm.equalsIgnoreCase("disable") || commandTerm.equalsIgnoreCase("0")) {
+                if (CafeBot.getGuildHandler().getCustomGuild(event.getGuild()).setDailyChannelID("0")) {
+                    event.getChannel().sendMessageEmbeds(CafeBot.getGeneralHelper().successEmbed(
+                            "Removed Daily Channel",
+                            "Successfully removed the daily channel."
+                    )).queue();
+                    return;
+                }
+                event.getChannel().sendMessageEmbeds(CafeBot.getGeneralHelper().sqlServerError()).queue();
+                return;
+            }
+
+            event.getChannel().sendMessageEmbeds(CafeBot.getGeneralHelper().errorEmbed(
+                    "Incorrect Extra Term",
+                    "You can run this command without extra arguments. You had the extra argument `" + commandTerm + "`. " +
+                            "The available command terms for this command are `disable`, `remove`, and `0`."
+            )).queue();
+            return;
+        }
+
+        if (CafeBot.getGuildHandler().getCustomGuild(event.getGuild()).setDailyChannelID(event.getChannel().getId())) {
+            event.getChannel().sendMessageEmbeds(CafeBot.getGeneralHelper().successEmbed(
+                    "Updated Daily Channel",
+                    "Successfully set the daily channel to this channel. " +
+                            "To remove the daily channel, just delete the channel. Just a reminder, " +
+                            "any integrations you have with this channel will be deleted upon the copy of this channel."
+            )).queue();
+
+            return;
+        }
+        event.getChannel().sendMessageEmbeds(CafeBot.getGeneralHelper().sqlServerError()).queue();
+    }
+
+    @Override
+    public String getName() {
+        return "set-daily-channel";
+    }
+
+    @Override
+    public ArrayList<String> getAliases() {
+        ArrayList<String> arrayList = new ArrayList<>();
+        arrayList.add("setdailychannel");
+        arrayList.add("setdaily");
+        arrayList.add("set-daily");
+        return arrayList;
+    }
+
+    @Override
+    public String getDescription() {
+        return "Sets the current channel to the daily channel. This channel will be deleted and re-added every day, once a day.";
+    }
+
+    @Override
+    public String exampleUsage(String prefix) {
+        return "`" + prefix + "set-daily-channel` or `" + prefix + "set-daily-channel remove`";
+    }
+
+    @Override
+    public Usage getUsage() {
+        Usage usage = new Usage();
+        usage.addUsage(CommandType.TEXT, "Disable/Remove/0", false);
+        return usage;
+    }
+
+    @Override
+    public CategoryType getCategoryType() {
+        return CategoryType.MODERATION;
+    }
+}
```

Comment:
Somewhere in here, make sure the channel is also not being used by other channels. For example, it can also not be used by the venting channel.